osu.Game.Rulesets.Catch/Skinning/Legacy/LegacyCatchHitExplosion.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Rulesets.Catch.Objects;
+using osu.Game.Rulesets.Catch.UI;
+using osu.Game.Skinning;
+using osuTK;
+
+namespace osu.Game.Rulesets.Catch.Skinning.Legacy
+{
+    public class LegacyCatchHitExplosion : CatchHitExplosion
+    {
+        private readonly Sprite explosion1;
+        private readonly Sprite explosion2;
+
+        /// <inheritdoc cref="Catcher.ALLOWED_CATCH_RANGE"/>
```

Comment:
We don't really use `inheritdoc` anywhere else, and this one is a touch misleading (mentions "The width of the catcher which can receive fruit", which is inaccurate in this context given the value).